content/components/checkbox-group.mdx

```diff
@@ -2,7 +2,7 @@
 title: Checkbox group
 description: Checkbox group renders a set of checkboxes.
 reactId: checkbox_group
-figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=15341%3A47574
+figmaId: checkbox
```

Comment:
Is it intentional to document `checkbox` in `checkbox_group` for Figma?

Reply:
```suggestion
```